ana_scans.py

```diff
@@ -217,13 +224,20 @@ def launchAnaArgs(anaType, cName, cType, scandate, scandatetrim=None, ztrim=4.0,
       print("Normal termination")
       pool.close()
       pool.join()
+      print "Results:", res.get()
+      for returncode in res.get():
+        if returncode != 0:
+          sys.exit(returncode)
+        pass
     except KeyboardInterrupt:
       print("Caught KeyboardInterrupt, terminating workers")
       pool.terminate()
```

Comment:
Why no `sys.exit(-1)` here?  Or, why present in the other `except` blocks?

Reply:
I used `sys.exit(-1)` to indicate an exception in `ana_scans.py`. When `ana_scans.py` works properly, the exit code of (the first failing) subcommands is just forwarded to the caller.

Reply:
OK, so you're not interested in a `-1` when you get a `KeyboardInterrupt`?

Reply:
Not for unit tests, but that's inconsistent behavior